src/index.html

```diff
@@ -4,9 +4,398 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#body" class="logo">
+        <img
+          src="/images/logo.png"
+          alt="logo"
+          class="logo__image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#body" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#body" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#body" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#body"
+            class="nav__link"
+            data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a href="#body"
              class="nav__link"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```
fix indent. check other places